stl/inc/xpolymorphic_allocator.h

```diff
@@ -274,6 +273,11 @@ namespace pmr {
             _Uses_allocator_construct(_Ptr, _Al, *this, _STD forward<_Types>(_Args)...);
         }
 
+        template <class _Uty>
+        _CXX20_DEPRECATE_POLYMORPHIC_ALLOCATOR_DESTROY void destroy(_Uty* const _Ptr) noexcept /* strengthened */ {

```

Comment:
Did you actually implement it to deprecate it?

Reply:
Yes. After holding out hope forever that LWG would just strike the silly thing.